django_graph_api/tests/test_errors.py

```diff
@@ -0,0 +1,73 @@
+from test_app.schema import schema
+
+
+def test_blank_query(starwars_data):
+    query = ''
+    assert schema.execute(query) == {
+        'errors': [
+            {
+                'message': 'Parse error: Unexpected end of input'
+            }
+        ]
+    }
+
+
+def test_non_existent_episode(starwars_data):
+    document = '''
+        {
+            episode (number: 12) {
+                name
+            }
+        }
+        '''
+    assert schema.execute(document) == {
+        "data": {
+            "episode": None
+        },
+        "errors": [
+            {
+                "message": "Error resolving episode: Episode matching query does not exist."
+            }
+        ]
+    }
+
+
+def test_non_existent_field(starwars_data):
+    document = '''
+        {
+            episode (number: 4) {
+                name
+                other_field
+            }
+        }
+        '''
+    assert schema.execute(document) == {
+        "data": {
+            "episode": {
+                "name": "A New Hope",
+                "other_field": None
+            }
+        },
+        "errors": [
+            {
+                "message": "Episode does not have field other_field"
+            }
+        ]
+    }
+
+
+def test_multiple_queries(starwars_data):
+    document = '''
+        mutation MyMutation {
+          episodes {
+            name
+          }
+        }
+        '''
```

Comment:
This fwiw is testing what happens when zero queries are defined (which is currently same behavior as multiple queries).

Reply:
Oh yeah, I forgot to change the name 👍